__tests__/app.test.js

```diff
@@ -16,3 +23,31 @@ describe.only("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: responds with an array of topic objects, each containing slug and description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toBeInstanceOf(Array);
+
+        body.topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  test("404: responds with an error if the topic is not found", () => {
```

Comment:
This 404 is not unique to the topics endpoint, this is just an endpoint that doesn't exist. While this is something that absolutely should be tested, it should be in it's own describe block